iati/core/schemas.py

```diff
@@ -24,15 +24,15 @@ class Schema(object):
 
     """
 
-    def __init__(self, name=None):
+    root_element_name = ''
```

Comment:
I'm confident this is at the right scope (and also prevents the defining  an `__init__` just to set `self.root_element_name` and then call `super()`).

I'm not sure if this means it's a constant though?